safe/processors/post_processors.py

```diff
@@ -79,6 +85,87 @@
     }
 }
 
+post_processor_distance = {
+    'key': 'post_processor_distance',
+    'name': tr('Distance Post Processor'),
+    'description': tr(
+        u'A post processor to calculate the distance between two points. The '
+        u'unit is defined in the exposure definition.'),
```

Comment:
Hum, does place has a unit? Because we use a place exposure, right?

Reply:
Leftover from copy and paste :) forget to clean it up.